perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -103,11 +107,11 @@ def _RunPing(sending_vm, receiving_vm, receiving_ip, ip_type):
   Returns:
     A list of samples, with one sample for each metric.
   """
-  if not sending_vm.IsReachable(receiving_vm):
+  if ip_type == vm_util.IpAddressSubset.INTERNAL and not sending_vm.IsReachable(receiving_vm):
     logging.warn('%s is not reachable from %s', receiving_vm, sending_vm)
     return []
 
-  logging.info('Ping results (ip_type = %s):', ip_type)
+  logging.info('Ping results (ip_type = %s):' % ip_type)
```

Comment:
Please revert this change (we have a different internal linter that complains about `logging-not-lazy`)  We're working to align both of them...

Reply:
change reverted